### PR TITLE
Westinm/clean build

### PR DESCRIPTION
**Pull request by westin-m**
Id Web's build process is split into many `.yaml` files. Reduce this number while maintaining the build process and its organization. See the provided diagram for a summary of this. 
id web build pipeline

### CODE REVIEW

File: build/template-install-dependencies.yaml [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 6'
+  inputs:
+    version: 6.0.x   
+
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 7'
+  inputs:
+    version: 7.0.x   
+
```
Comment (jennyf19) [inline build/template-install-dependencies.yaml:20]: no .net 8, or is that part of the conditional compilation PR?

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: build/template-install-dependencies.yaml (hunk 1 above)
Comment (westin-m) [reply build/template-install-dependencies.yaml:20]: That's part of the other PR

Comment (jennyf19) [reply build/template-install-dependencies.yaml:20]: Yep, saw it.

**Review by jmprieur — APPROVED** [review]
LGTM

**Pull request merged**